### PR TITLE
Fix Windows paths resulting from `get_attached_file`.

### PR DESCRIPTION
**Pull request by dream-encode**
Trac ticket: https://core.trac.wordpress.org/ticket/36308.

### CODE REVIEW

File: tests/phpunit/tests/post.php [hunk 1]
```diff
@@ -1797,4 +1797,40 @@ public function test_unstick_post_with_non_sticky_post_id_does_not_update_option
 		unstick_post( 3 );
 		$this->assertSameSets( array( 1, 2, 2 ), get_option( 'sticky_posts' ) );
 	}
+
+	/**
+	 * Testing the wp_get_attached_file() function.
+	 *
```
Comment (costdev) [inline tests/phpunit/tests/post.php]:
```suggestion
	 * @covers ::wp_get_attached_file
```

**Review by SergeyBiryukov — APPROVED** [review]

File: tests/phpunit/tests/post/getAttachedFile.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+/**
+ * @group post
+ * @covers ::get_attached_file
+ */
+class Tests_Post_GetAttachedFile extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 36308
+	 */
+	public function test_get_attached_file_with_windows_paths() {
```
Comment (costdev) [inline tests/phpunit/tests/post/getAttachedFile.php:12]: Using a data provider for the `file`, `expected` values, and possibly the `message` value, will ensure that both assertions run even if one happens to fail, and makes it easier to add more tests for this in future if needed.

**Review by costdev — CHANGES_REQUESTED** [review]
Just one suggestion to use a data provider for the new test method.

**Review by johnbillion — CHANGES_REQUESTED** [review]
This looks good and there's a good amount of test coverage, but I'd also like to see the tests in `Tests_Post_GetAttachedFile` use a data provider 👍

**Comment by costdev** [comment]
> This looks good and there's a good amount of test coverage, but I'd also like to see the tests in `Tests_Post_GetAttachedFile` use a data provider 👍 

Hi @johnbillion, see [PR 3049](https://github.com/WordPress/wordpress-develop/pull/3049) which is a refresh of this PR that uses a data provider in `Test_Post_GetAttachedFile`

**Comment by johnbillion** [comment]
Closing in favour of #3049 👍

**Pull request closed without merging**